pcdsdevices/epics_motor.py

```diff
@@ -1299,6 +1299,18 @@ class BeckhoffAxisEPS(BeckhoffAxis):
               doc='PLC error handling, aux functions, and EPS.')
 
 
+class BeckhoffAxisEPSCustom(BeckhoffAxis):
+    """
+    A beckhoff axis where custom EPS Logic has been implemented in
+    a PLC and needs a custom screen to display EPS information
+
+    EPS screens are found in
+    reg/g/pcds/epics-dev/screens/pydm/eps_screens/${beamline}/${name}
```

Comment:
I think this is a reasonable place for these screens, but a few things to note:

- This forces us to always use epics-dev in this version of the code. In the future I intend for people to be able to tag and release their screens, so this will need to be revised in some way in a later revision.
- We should make sure there's a reasonable readme in this directory that explains what it is for.
- We should stop proliferating the deprecated `/reg` paths, preferring the newer `/cds` paths instead (and, we should always have the preceding `/` for correctness.

```suggestion
    /cds/group/pcds/epics-dev/screens/pydm/eps_screens/${beamline}/${name}
```